data/json/snippets.json

```diff
@@ -79,6 +79,11 @@
     "category": "flier",
     "text": [ "This is an advertisement for Sybil's Cyber Boutique.  It shows a picture of several bikini-clad female doctors removing the skin from man's face.  The patient is strapped to a chair but cheerfully gives two thumbs up.  The caption reads: \"Free Plastic Surgery While You Wait!\"" ]
   },
+  {
+    "type": "snippet",
+    "category": "flier",
+    "text": [ "This is an advertisement for Sybil's Cyber Boutique.  It shows a picture of several bikini-clad nurses giving medicine to a delighted patient, who sucks it with a neede pointing out of his finger.  Outside the window two shadow figures are doing the same with something less resembling medicine.  The caption reads: \"Why use old IV when you can have Intravenous Needle CBM!\"" ]
```

Comment:
> several bikini-clad nurses giving medicine to a delighted patient, who sucks it with a neede pointing out of his finger

`needle`

Also, he sucks what, exactly? 

> Outside the window two shadow figures are doing the same with something less resembling medicine

Two shadow figures are... providing medicine to a patient? Sorry, this third sentence confuses me a bit